Arduino Code/Arduino Libraries/HanReader/src/HanReader.cpp

```diff
@@ -5,15 +5,8 @@ HanReader::HanReader()
   
 }
 
-void HanReader::setup(HardwareSerial *hanPort, unsigned long baudrate, SerialConfig config, Stream *debugPort)
+void HanReader::setup(HardwareSerial *hanPort, Stream *debugPort)
 {
-	// Initialize H/W serial port for MBus communication
-	if (hanPort != NULL)
```

Comment:
Har flyttet ansvaret for aa aapne Serial porten ut av HanReader og inn i setup() i sketch'en. Dette fordi SerialConfig ikke lengre kompilerer og fordi ansvaret bedre hoerer hjemme der.